examples/resources/btp_subaccount_environment_instance/resource.tf

```diff
@@ -35,3 +35,26 @@ resource "btp_subaccount_environment_instance" "cloudfoundry" {
     delete = "30m"
   }
 }
+
+# creates a kyma environment in a given account
+# NOTE: for the available parameter values, check https://help.sap.com/docs/btp/sap-business-technology-platform/available-plans-in-kyma-environment
+resource "btp_subaccount_environment_instance" "kyma" {
+  subaccount_id    = "6aa64c2f-38c1-49a9-b2e8-cf9fea769b7f"
```

Comment:
is this a real subaccount id? or was it made up for the documentation?

Reply:
Made up for docu